client/apollo/js/main.js

```diff
@@ -931,12 +931,7 @@ return declare( [JBPlugin, HelpMixin,Evented],
                     locationBox.set('value',locationVal,false);
                 }
                 else{
-                    var locationBoxString = Util.assembleLocStringWithLength( currRegion );
-                    if(currRegion.ref.startsWith("{")){
-                        sequenceString = locationBoxString.substring(0,locationBoxString.lastIndexOf("}")+1);
-                        sequenceObject = JSON.parse(sequenceString).sequenceList[0];
-                        locationBoxString = sequenceObject.name + locationBoxString.substr(locationBoxString.lastIndexOf(":"));
-                    }
+                    var locationBoxString = Util.assembleProjectedString(currRegion);
```

Comment:
There is no function `assembleProjectedString` in `JBrowse/Util.js`

I get this error on the console when clicking anywhere on the browser:
```
main.js:961 Uncaught TypeError: Util.assembleProjectedString is not a function
    at main.js:961
    at Evented.<anonymous> (lang.js:375)
    at Evented.dispatcher [as on/jbrowse/v1/n/navigate] (aspect.js:101)
    at Function.on.emit (on.js:309)
    at Function.on.emit (on.js:350)
    at Evented.emit (Evented.js:32)
    at Object.publish (topic.js:25)
    at Object.publish (Browser.js:1877)
    at Object.onCoarseMove (Browser.js:2896)
    at Object.advice (aspect.js:114)
```